crates/astria-composer/src/composer_status.rs

```diff
@@ -0,0 +1,20 @@
+/// Announces the current status of the Composer for other modules in the crate to use
+#[derive(Debug, Default)]
+pub(super) struct ComposerStatus {
```

Comment:
Please move this type into `composer` and refer to it as `composer::Status`.

Reply:
sure will do